.gitignore

```diff
@@ -1,3 +1,6 @@
 *.o
 *.d
 ttt
+train
+state_value.bin
```

Comment:
Due to the possibility of the file name being changed, and the likelihood that we probably wouldn't want to include any binary files in the source code, using ```*.bin``` might be better than hardcoding the file name.